### PR TITLE
fix: expand all uses of spec to specification and docs to documentation

### PR DESCRIPTION
**Pull request by JakeChampion**
during an external accessibility audit it was mentioned that the abbreviation 'spec' and 'docs' may not be well known and it would be best to use the full words instead.

I've updated all uses of `spec` and `docs` - including in url paths that we own

resolves https://github.com/Financial-Times/origami/issues/550

### CODE REVIEW

File: apps/website/pages/manifests.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 description: Origami manifest files are used to describe and configure Origami projects.
-permalink: /docs/manifests/
+permalink: /documentation/manifests/
 layout: collection
 collection_id: manifests
 collection_title: Manifests
 
 # Redirect from legacy URLs
 redirect_from:
+  - /docs/manifests/
```
Comment (notlee) [inline apps/website/pages/manifests.md:11]: 👍

File: components/o-colors/package.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
     "@financial-times/o-normalise": "^3.2.0",
     "@financial-times/o-overlay": "^4.2.1",
     "@financial-times/o-syntax-highlight": "^4.2.0",
-    "@financial-times/o-tabs": "^6.2.0"
+    "@financial-times/o-tabs": "^7.0.1"
```
Comment (notlee) [inline components/o-colors/package.json:40]: 👀

Comment (notlee) [reply components/o-colors/package.json:40]: Do demos markup need updating to migrate to o-tabs@v7?

Comment (JakeChampion) [reply components/o-colors/package.json:40]: Yep, the demos already use the v7 markup but they didn't pull in the right version of o-tabs. this fixes that :-)

release notes for o-tabs v7 -- https://github.com/Financial-Times/origami/releases/tag/o-tabs-v7.0.0

**Review by notlee — APPROVED** [review]

**Pull request merged**